triveni-codev/GBWebApp-dev
# Darlp certificate requests

Hi @triveni-codev ,

This is already good. Take note there are 3 tickets included here. See below further details.

PRIMARY ASANA TICKET: https://app.asana.com/0/1201943683850147/1205688419493096

To Test:
  -should be in dynamic table implementation
  -all major dynamic table buttons (add, edit, delete and view) should be functional
  -should already be using the associated certificate request model and it's corresponding service
  -should now be able to commit / revert any changes

Don't Test (not implemented):
  -export button wasn't implemented since before
  -download file (not implemented as well)
  
  
  
SECONDARY ASANA TICKET: https://app.asana.com/0/1201943683850147/1205734104968678

To Test: Merely check if the dropdown selection for adding new Network should only include ATSC-3, ATSC-Cable and ATSC-Terrestrial (all other types are removed)

TERTIARTY ASANA TICKET: https://app.asana.com/0/1201943683850147/1205734104968681

To Test: Just check if you can add/edit a new/existing channel extension or service extension (make sure you have the updated GBOne version)

Don't Test: Don't include the Broadcast Service Icon since it's for the other ticket
